src/plugins/github/parseReferences.js

```diff
@@ -46,20 +41,5 @@ export function findGithubUrlReferences(body: string): GithubUrlMatch[] {
       /(?:\W|$)/.source,
     "gm"
   );
-  return findAllMatches(urlRegex, body).map((match) => {
-    let commentFragment: $ElementType<GithubUrlMatch, "commentFragment">;
-    if (match[5] != null) {
-      // we found a comment fragment
-      commentFragment = {fragmentType: match[6], fragmentNumber: +match[7]};
-    } else {
-      commentFragment = null;
-    }
-    return {
-      repoOwner: match[1],
-      repoName: match[2],
-      parentType: match[3],
-      number: +match[4],
-      commentFragment,
-    };
-  });
+  return findAllMatches(urlRegex, body).map((match) => match[0].trim());
```

Comment:
This is totally fine with me. You could also use
```js
return body.match(urlRegex).map((x) => x.trim());
```
because you do not care about any particular capturing groups.

Reply:
I tried that, but body.match(urlRegex) returns null and so .map throws a type error. Leaving unchanged.

Reply:
Apparently `match` returns an array of matches, or `null` if there are none—a cardinal sin. Your way is better. :-)